applications/application-template.md

```diff
@@ -122,7 +122,7 @@ To assist you in defining it, we have created a document with examples for some
 
 * **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
 * **Full-Time Equivalent (FTE):**  Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE)
-* **Total Costs:** Amount of payment in USD for the whole project. The maximum amount of funding depends on the [level](../README.md#level_slider-level) of your grant application. 
+* **Total Costs:** Requested amount in USD for the whole project (e.g. 12,000 USD). Note that the acceptance criteria and additional benefits vary depending on the [level](../README.md#level_slider-levels) of funding requested. This and the costs for each milestone need to be provided in USD; if the grant is paid out in Bitcoin, the amount will be calculated according to the exchange rate at the time of payment.
```

Comment:
I missed whether or not we want to allow teams to apply for a higher level than their funding requires, in my opinion this is additional complexity that we don't really need. If a team want the benefits of level 2 I think it's fair to ask that they deliver something at least worth $10k, and if they really want VC introductions they can just ask.